app/leave-app/backend/constants.bal

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2024 WSO2 LLC. (http://www.wso2.org).
```

Comment:
```suggestion
// Copyright (c) 2024 WSO2 LLC. (https://www.wso2.org).
```

Reply:
@Rashmika998 Can you check whether this should be `https://www.wso2.com` ?

Reply:
I've checked with [API Manager code files](https://github.com/wso2/product-apim/blob/master/modules/integration/tests-common/admin-clients/src/main/java/org/wso2/am/admin/clients/application/ApplicationManagementClient.java#L2) license header and it also contains `http`

Reply:
@chanukaranaba @jsdjayanga

Reply:
This is a 3 year old code. Can we get a License file of a new code? @Rashmika998

Reply:
Yeah. Seems like they are not updating it. I'll add the latest one.

Reply:
@suhand @chanukaranaba I had a chat with APIM team and they confirmed that both options are valid. In their codes, they use both links.